### PR TITLE
chore: Update README.md for release version 0.2.0

### PR DESCRIPTION
**Pull request by dvandra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,16 +59,30 @@ File(Code on macOS) > Preferences > Settings to open your [Settings](https://cod
...
-![ screencast ](images/0.1.0/multi-stack-analysis.gif)
+![ screencast ](images/0.2.0/multi-stack-analysis.gif)
 
 ---
 
 **Note** It creates a folder `target` in workspace which is used for processing of manifest files, needed for generating stack report. So kindly add `target` in `.gitignore`.
 
+## Register for a free Snyk Account and Connect Snyk to your Red Hat Dependency Analytics
+
+1. By clicking on the `Sign up for a free Snyk account` from 'Dependency Analytics report' goes to the Snyk sign up page for a free Snyk account. Just after signing up for a free Snyk account it goes to the 'Snyk's Landing page' which shows `Snyk token` to connect Snyk with your Red Hat Dependency Analytics. Copy and paste the Snyk token into Red Hat Dependency Analytics Report as shows below.
```
Comment (prashbnair) [inline README.md:74]:
```suggestion
1. Clicking on the `Sign up for a free Snyk account` from 'Dependency Analytics report' will take you to the Snyk sign up page for a free Snyk account. After signing up for a free Snyk account it goes to the 'Snyk's Landing page' which shows `Snyk token` to connect Snyk with your Red Hat Dependency Analytics. Copy and paste the Snyk token into Red Hat Dependency Analytics Report as shows below.
```

File: README.md [hunk 2]
```diff
@@ -59,16 +59,30 @@ File(Code on macOS) > Preferences > Settings to open your [Settings](https://cod
...
+
+![ screencast ](images/0.2.0/snyk-sign-up.gif)
+
+2. Look for ![snyk button](images/0.2.0/snyk-button.png) in Dependency Analytics Report and click on the button to enter your Snyk Token. Paste your snyk token and click on the `Submit button`.
+
+![ screencast ](images/0.2.0/snyk-token.gif)
+
+3. After successfully entering the Snyk token the Dependency Analyitcs report will updated with the detailed information of security vulnerability unique to snyk and vulnerabilities having publicly known exploit.
```
Comment (prashbnair) [inline README.md:82]:
```suggestion
3. After successfully entering the Snyk token the Dependency Analyitcs report will be updated with detailed information about security vulnerabilities unique to Snyk and vulnerabilities having publicly known exploits.
```

**Review by prashbnair — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -59,16 +59,30 @@ File(Code on macOS) > Preferences > Settings to open your [Settings](https://cod
...
-![ screencast ](images/0.1.0/multi-stack-analysis.gif)
+![ screencast ](images/0.2.0/multi-stack-analysis.gif)
 
 ---
 
 **Note** It creates a folder `target` in workspace which is used for processing of manifest files, needed for generating stack report. So kindly add `target` in `.gitignore`.
 
+## Register for a free Snyk Account and Connect Snyk to your Red Hat Dependency Analytics
+
+1. Clicking on the `Sign up for a free Snyk account` from 'Dependency Analytics report' will take you to the Snyk sign up page for a free Snyk account. After signing up for a free Snyk account it goes to the 'Snyk's Landing page' which shows `Snyk token` to connect Snyk with your Red Hat Dependency Analytics. Copy and paste the Snyk token into Red Hat Dependency Analytics Report as shows below.
```
Comment (prashbnair) [inline README.md:74]:
```suggestion
1. Clicking on the `Sign up for a free Snyk account` from 'Dependency Analytics report' will take you to the Snyk sign up page for a free Snyk account. After signing up for a free Snyk account it goes to the 'Snyk's Landing page' which shows `Snyk token` to connect Snyk with your Red Hat Dependency Analytics. Copy and paste the Snyk token into Red Hat Dependency Analytics Report as shown below.
```

**Review by prashbnair — APPROVED** [review]

**Review by mathur07 — APPROVED** [review]

**Comment by mathur07** [comment]
@dvandra are we updating the CA gif with new shortcut icon or it fine?

**Comment by dvandra** [comment]
> @dvandra are we updating the CA gif with new shortcut icon or its fine?

@mathur07 I have just updated the gif with new shortcut icon.
PTAL

**Review by mathur07 — APPROVED** [review]
LGTM

**Review by arajkumar — CHANGES_REQUESTED** [review]
@dvandra Some of the screenshots has to be updated with latest "Button text" #423

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**